### PR TITLE
readme: add list of known mods

### PR DESCRIPTION
**Pull request by mewmew**
Updates #616. (leaving the issue open for a while longer, so the list becomes known).

### CODE REVIEW

File: docs/mods.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Mods
+
+This is a list of known mods to DevilutionX. Feel free to extend it by sending a PR.
+
+## Infernity
```
Comment (AJenbo) [inline docs/mods.md]: Infernity is technically a Devilution mod and not DevilutionX, and as it's a rather old base it's probably hard to pick pices from it. Still worth having on the list.

Comment (mewmew) [reply docs/mods.md]: Didn't know that. @qndel, rebase! :)

Comment (AJenbo) [reply docs/mods.md]: He is waiting for me to rewrite all the code first :D

Comment (mewmew) [reply docs/mods.md]: lol

Comment (qndel) [reply docs/mods.md]: infernity's biggest feature were custom resolutions :) everything else is minor compared to that and I could port it to devilutionx in a day

**Comment by mewmew** [comment]
@AJenbo and @qndel, this PR is now ready for merge. Does it look fine to you?

**Comment by qndel** [comment]
LGTM

**Comment by galaxyhaxz** [comment]
Didn't know so many mods already existed. Really cool stuff from footballhead.

Oh, you forgot the mod that puts Farnham on a flying carpet and turns Diablo into a bacon cheeseburger. It's called `Diablo the 3L7te`

**Comment by AJenbo** [comment]
GKD350h, RG-350 handheld and RetroFW have been merged so technically they are no longer ports :)

**Comment by mewmew** [comment]
> GKD350h, RG-350 handheld and RetroFW have been merged so technically they are no longer ports :)

Good to know! We should have a section somewhere then with *cool screenshots*! That's really the main purpose I wanted to add `mods.md` :p

**Comment by mewmew** [comment]
> GKD350h, RG-350 handheld and RetroFW have been merged so technically they are no longer ports :)

Do we have PlayStation Vita support?

**Comment by AJenbo** [comment]
Yeah a screen shot section would be really nice, I removed the wo original as they where missleading.

Vita, android, iOS and 3ds are still unmerged and with a mention

**Pull request merged**